crates/extensions_ui/src/extension_suggest.rs

```diff
@@ -70,6 +70,7 @@ const SUGGESTIONS_BY_EXTENSION_ID: &[(&str, &[&str])] = &[
     ("templ", &["templ"]),
     ("terraform", &["tf", "tfvars", "hcl"]),
     ("toml", &["Cargo.lock", "toml"]),
+    ("typst", &["typ"]),
```

Comment:
https://github.com/WeetHet/typst.zed/blob/91781bb3fa0340655d756feb8edd7067f3fc357c/languages/typst/config.toml#L3 mentions another path suffix, any reason to omit it here?

Reply:
Oh, maybe it's vice-versa, thank you for pointing out.